presto-docs/src/main/sphinx/release/release-0.199.rst

```diff
@@ -0,0 +1,75 @@
+* Add support for yielding when unspilling an aggregation.
+* Expand grouped execution support to ``GROUP BY`` and ``UNION ALL``, making it possible
+  to execute aggregations with less peak memory usage.
+* Change the signature of ``round(x, d)`` and ``truncate(x, d)`` functions so that
+  ``d`` is of type ``INTEGER``. Previously, ``d`` could be of type ``BIGINT``.
+  This behavior can be reverted with the ``deprecated.legacy-round-n-bigint`` config option
+  or the ``legacy_round_n_bigint`` session property.
+* Optimize the :func:`ST_Intersection` function for rectangles aligned with coordinate axes
+  (e.g., polygons produced by the :func:`ST_Envelope` and :func:`bing_tile_polygon` functions).
+* Finish joins early when possible if one side has no rows. This happens for
+  either side of an inner join, for the left side of a left join, and for the
+  right side of a right join.
+* Improve predicate evaluation performance during predicate pushdown in planning.
+* Improve the performance of queries that use ``LIKE`` predicates on the columns of ``information_schema`` tables.
+* Improve the performance of map-to-map cast.
+* Improve the performance of :func:`ST_Touches`, :func:`ST_Within`, :func:`ST_Overlaps`, :func:`ST_Disjoint`,
+  and :func:`ST_Crosses` functions.
+* Improve the serialization performance of geometry values.
+
+Server RPM Changes
+------------------
+
+* Add support for installing Presto server RPM to machines with OpenJDK.
+
+Security Changes
+----------------
+
+* Add support for authentication with JWT access token.
+
+Hive Changes
+------------
+
+* Fix ORC writer failure when writing ``NULL`` values into columns of type ``ROW``, ``MAP``,  or ``ARRAY``.
+* Fix ORC writers incorrectly writing non-null values as ``NULL`` for all types.
+* Support reading Hive partitions that have a different bucket count than the table,
+  as long as the ratio is a power of two (``1:2^n`` or ``2^n:1``).
+* Add support for the ``skip.header.line.count`` table property.
+* Prevent reading from tables with the ``skip.footer.line.count`` table property.
+* Partitioned tables now have a hidden system table that contains the partition values.
+  A table named ``example`` will have a partitions table named ``example$partitions``.
+  This provides the same functionality and data as ``SHOW PARTITIONS``.
+* Partition name listings, both via the ``$partitions`` table and using
+  ``SHOW PARTITIONS``, are no longer subject to the limit defined by the
+  ``hive.max-partitions-per-scan`` config option.
+
+Thrift Connector Changes
+------------------------
+
+* Most of the config property names are different due to replacing the
+  underlying Thrift client implementation. Please see :doc:`/connector/thrift`
+  for details on the new properties.
+
+SPI Changes
+-----------
+
+* Introduce ``SystemTablesProvider`` interface, complementing `Connector#getSystemTables` method,
```

Comment:
This description is bad because `SystemTablesProvider` is an implementation detail of the engine, not part of the SPI. Let's change this to
```
* Allow connectors to provide system tables dynamically.